nss-evening-cohort-29/flamingo-gitsub
# added pinning functionality within repos page

## Description
added pinning functionality within the repos page. had to do it within that page since clicking overview reloads the page and the push to array data is lost in doing so

## Related Issue
#69 

## Motivation and Context
instructions said users should be able to pin repos

## How Can This Be Tested?
click pin on a repo in the repos page and it should go to the top and have a star. should also be able to do this to newly created repos via the form on the repos page.

## Types of changes
